packages/aws-cdk/test/api/exec.test.ts

```diff
@@ -0,0 +1,126 @@
+import * as cxschema from '@aws-cdk/cloud-assembly-schema';
+import * as cdk from '@aws-cdk/core';
+import * as fs from 'fs';
+import * as os from 'os';
+import * as path from 'path';
+import * as semver from 'semver';
+import { ImportMock } from 'ts-mock-imports';
+import { SdkProvider } from '../../lib';
+import { execProgram } from '../../lib/api/cxapp/exec';
+import { Configuration, Settings } from '../../lib/settings';
+
+// We need to increase the default 5s jest
+// timeout for async tests because the 'execProgram' invocation
+// might take a while :\
+const TEN_SECOND_TIMEOUT = 10000;
+
```

Comment:
Minor: can we switch to 2 space indent? From what I can see, the aws-cdk package is currently using that and let's not mix it up.

In general, I think we should all switch to 2 space indent.

Reply:
👍